README.md

```diff
@@ -12,6 +12,10 @@
 [![All Contributors](https://img.shields.io/badge/all_contributors-2-orange.svg?style=flat-square)](#contributors-)
 <!-- ALL-CONTRIBUTORS-BADGE:END -->
 
+![Pycrostates_logo](./docs/_static/img/Pycrostates_logo_black.png#gh-light-mode-only)
+
+![Pycrostates_logo](./docs/_static/img/Pycrostates_logo_white.png#gh-dark-mode-only)
+
```

Comment:
We need to put an absolute link here, or it will not render on Pypi.
```suggestion
![Pycrostates_logo](https://raw.githubusercontent.com/vferat/pycrostates/main/docs/_static/img/Pycrostates_logo_black.png#gh-light-mode-only)

![Pycrostates_logo](https://raw.githubusercontent.com/vferat/pycrostates/main/docs/_static/img/Pycrostates_logo_white.png#gh-dark-mode-only)

```

Reply:
Also Pypi will not be able to handle the` #gh-dark-mode-only` tags and will render the two images.
So we have two options:
-  to have light/dark versions on GitHub and 2 images on Pypi
-  to have one light image on GitHub and one light image on Pypi.

I am more in favour for the 1rst option has there is more traffic on GitHub.

Reply:
Good point. I think we can get around with HTML tags, see [this](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#specifying-the-theme-an-image-is-shown-to) (and the `#gh-dark-mode-only` and `gh-light-mode-only` are deprecated).

Reply:
To be tested on [Test Pypi](https://test.pypi.org/).